guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -122,15 +122,15 @@ _990_
 ifndef::foreman-deb[]
 ----
 # {package-install} nfs-utils
-# systemctl enable rpcbind nfs-server
-# systemctl start rpcbind nfs-server nfs-lock nfs-idmapd
+# systemctl enable --now rpcbind nfs-server
+# systemctl start nfs-lock nfs-idmapd
 ----
 endif::[]
 ifdef::foreman-deb[]
 ----
 # {package-install} nfs-kernel-server nfs-common
```

Comment:
`nfs-kernel-server` depends on `nfs-common`, so I see no good reason to explicitly call that out? (scope creep, sorry)